portfolio/src/main/webapp/index.html

```diff
@@ -132,10 +132,44 @@ <h3>Beadwork</h3>
         <button onclick="addRandomFact()">Want a fact about me?</button>
         <div id="greeting-container"></div>
 
-        <h1>Comments</h1>
-        <p>Click here to see a random comment:</p>
-        <button onclick="getRandomQuoteUsingArrowFunctions()">Comments!</button>
-        <div id="comment-container"></div>
+        <div id="allcomments">
+            <h3>Comments</h3>
+            <form>
+                <select id="language" name="language" onchange="getTranslation(value)">
+                    <option value="en">English</option>
+                    <option value="zh">Chinese</option>
+                    <option value="fr">French</option>
+                    <option value="hi">Hindi</option>
+                    <option value="ig">Igbo</option>
+                    <option value="es">Spanish</option>
+                    <option value="yo">Yoruba</option>
+                </select>
+            </form>
+            <!-- We will show past comments in the list below-->
+            <ul id="comment-container"></ul>
+            
+            <h5>Leave me a Comment</h5>
+            <!-- This is the form for entering a new comment -->
+            
+            <form action="/data" method="POST" id="commentform">
+            
```

Comment:
I guess we don't need this blank line here? or do you want to add a </br> instead?